websites/documentation/src/content/writing/release-notes.mdx

```diff
@@ -57,3 +57,15 @@ FooBar Intro Line: What this is good for (like description)
 A longer explanation here that is just shown on the details page.
 If there is no more, remove the marker above, too.
 ```
+
+## Available frontmatters:
+
+- `date` (date, required): A `YYYY-MM-DD` date format (not a text, it's parsed as an ISO date)
+- `orderHint` (number, optional): A number that indicates the order of release notes with the same release date in the RSS feed. Use numbers in ascending order for all the release notes you want to order.
```

Comment:
out of curiosity .. does orderhint supports decimal numbers ?

Reply:
No, only integers...so whole numbers :)